### PR TITLE
Updated paths to match the new dev.api.carforyou.io

### PR DESCRIPTION
**Pull request by skomarica**
V2 :)

### CODE REVIEW

**Comment by masone** [comment]
Run `npm run format` to auto-format the code and fix the failing linter :)

**Comment by skomarica** [comment]
Let's see now, I hope that IntelliJ doesn't add some cleverness :)

File: next/lib/api.js [hunk 1]
```diff
@@ -9,7 +9,13 @@ export function fetchModels(make) {
 }
 export async function fetchListings({ make, model, priceFrom, priceTo, sort }) {
   const json = await fetchPath(
-    urlWithParams("iss/listings", { make, model, priceFrom, priceTo, sort })
+    urlWithParams("iss/listings/search", {
```
Comment (skomarica) [inline next/lib/api.js]: Remove `/search` from the path, as `iss` already indicates we use the search service

**Comment by masone** [comment]
Thanks!

**Pull request merged**